### PR TITLE
Bug 1231929 - Add telemetry to Firefox Sync. r=thrills,michielbdejong…

### PR DESCRIPTION
**Pull request by ferjm**
…,yifan

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=d32f00e2d1433739cab5351785112b7dedcf2ceb

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7e1c5bcfa6c873325b3959b2b20afce3fd71fa55

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=872771485d9b1981bc175cefd92c2c3e624c9684

File: apps/system/js/sync_manager.js [hunk 1]
```diff
@@ -218,7 +219,11 @@
         case 'disable':
         case 'sync':
           try {
-            Service.request('SyncStateMachine:' + request.name).catch(e => {
+            Service.request('SyncStateMachine:' + request.name).then(() => {
+              return LazyLoader.load('/shared/js/sync/telemetry.js');
```
Comment (michielbdejong) [inline apps/system/js/sync_manager.js]: Would maybe be cleaner to do this all from /shared/js/sync/bootstrap.js, so it runs in the sync app and not in the system app, but I guess it doesn't matter so much.

Comment (ferjm) [inline apps/system/js/sync_manager.js]: We are trying to record three user actions: enable, disable and manual sync. From the sync app we don't know when these actions happen. We only know that a sync was requested, but we don't know if it was a manual request or the sync manager timer firing.

Comment (michielbdejong) [inline apps/system/js/sync_manager.js]: ok, got it

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=ddae1899773ec6ccdc0621939896dd9d5d7c460d

**Pull request merged**